net/portmapper/portmapper.go

```diff
@@ -834,15 +897,10 @@ func (c *Client) Probe(ctx context.Context) (res ProbeResult, err error) {
 			}
 			metricUPnPOK.Add(1)
 			c.logf("[v1] UPnP reply %+v, %q", meta, buf[:n])
+
+			// Store the UPnP response for later selection
+			upnpResponses = append(upnpResponses, meta)
```

Comment:
Thoughts on enforcing a max in case there is some UPnP speaker(s) that floods us?

Reply:
Good idea; I think 10 should be plenty 👍